qabelbox/src/main/res/values-de/strings.xml

```diff
@@ -1,6 +1,152 @@
+    <string name="contact_import_successfull">Kontakt importiert</string>
+    <string name="contact_list_empty">Sie haben noch keine Kontakte.\nBenutzen Sie den Knopf in der Ecke unten rechts um Kontakte zu importieren</string>
+    <string name="contact_name">Name des Kontakts</string>
+    <string name="contact_public_key">Öffentlicher Schlüssel</string>
+    <string name="create_account_email">Bitte geben Sie ihre E-Mail an. Mit dieser Adresse können Sie ihren Account zurücksetzen</string>
+    <string name="create_account_email_hint">Ihre E-Mail</string>
+    <string name="create_account_enter_all_data">Bitte geben Sie alle Daten ein um fortzufahren</string>
+    <string name="create_account_enter_name_infos">Bitte geben Sie einen Nutzernamen für ihren Qabelaccount an. Ihr Nutzername darf bis zu 30 Zeichen haben</string>
+    <string name="create_account_final">Ihr Qabelaccount ist jetzt verfügbar</string>
+    <string name="create_account_final_headline">Qabelaccount registriert</string>
+    <string name="create_account_login_infos">Bitte loggen Sie sich in ihren Qabelaccount ein</string>
+    <string name="create_account_maximum_32_chars">Die maximale länge des Accountnamens </string>
+    <string name="create_account_name_hint">Nutzername</string>
+    <string name="create_account_password1_hint">Passwort</string>
+    <string name="create_account_password2_hint">Passwort wiederholen</string>
+    <string name="create_account_password_infos">Passwort eingeben</string>
+    <string name="create_account_passwords_dont_match">Die eingebenen Passwörter stimmen nicht überein.</string>
+    <string name="create_account_reset_password_infos">Bitte geben Sie die zu Ihrem Account gehörige E-Mail-Addresse ein</string>
+    <string name="create_account_username_hint">Nutzername</string>
+    <string name="create_box_account">Registriere einen Qabel account</string>
+    <string name="create_folder">Ordner erstellen</string>
+    <string name="create_identity_already_exists">Diese Identität besteht bereits</string>
+    <string name="create_identity_create">Erstellen einer Identität</string>
+    <string name="create_identity_enter_all_data">Bitte füllen Sie alle Felder aus</string>
+    <string name="create_identity_enter_name">Bitte geben Sie einen Namen für die Identität ein, zum Beispiel \'Jana\' oder \'Forentroll\'</string>
+    <string name="create_identity_enter_name_hint">Alias der Identität</string>
+    <string name="create_identity_final">Sie können diese Identität nun zum Kommunizieren und hochladen von Dateien verwenden</string>
+    <string name="create_identity_final_create_identity">Ihre Identität ist nun verfügbar</string>
+    <string name="create_identity_import">Importieren einer Identität</string>
+    <string name="create_identity_select_security">Wählen sie den Grad der Verschleierung bei der Kommunikation mit anderen Nutzern. Ein höherer Grad verbraucht mehr Strom.</string>
```

Comment:
If this is used for explaining the drop ID generation, I would propose to focus on more needed traffic instead of energy consumption.